src/server/object_services/schemas/object_md_schema.js

```diff
@@ -77,7 +77,7 @@ module.exports = {
         upload_started: { objectid: true },
         create_time: { date: true },
         cache_last_valid_time: { date: true },
-
+        last_modified_time: { date: true },
```

Comment:
Can you please add a comment that explains this new meta-data property.
Also I think a comment for previous line's cache_last_valid_time will be good.
```suggestion
        // last_modified_time is an optional property that is set on cache buckets
        // to separate the times of creation of the cache object (create_time)
        // vs the "real" LastModifiedTime from the hub.
        last_modified_time: { date: true },
```